tasks/smoke-test/tests/storybook.spec.ts

```diff
@@ -172,7 +172,7 @@ storybookTest(
     await page.goto(STORYBOOK_URL)
 
     // Click Redwood link in left nav
-    await page.locator('css=[data-item-id=redwood--page]').click()
+    await page.locator('id=redwood--page').click()
```

Comment:
I don't know why I had to change this now 🤷

Reply:
I think I'm getting the same error in https://github.com/redwoodjs/redwood/pull/6226, but it doesn't happen to any renovate PRs. Weird

Reply:
Ah I take that back, was failing because I forgot to update a string here: https://github.com/redwoodjs/redwood/pull/6226/commits/82ec51210f066ac49972b11cc7c7723c93345a95

Reply:
I think it workes in CI, but fails locally without this change.